README.md

```diff
@@ -124,13 +124,13 @@ If you wish to manually build the documentation, you can use `conda` to do so.
 1. Create a `conda` environment to build the documentation.
 
    ```bash
-   conda env create -n accessibility-docs python=3.9
+   conda env create -n a11y-team-compass-docs python=3.9
```

Comment:
I renamed the conda environment here and in other places to help avoid confusion. I was thinking somebody might skim this and think that these are commands for building end-user-facing accessibility docs